src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -138,6 +130,22 @@ AutomatedRssDownloader::~AutomatedRssDownloader()
     delete m_episodeRegex;
 }
 
+void AutomatedRssDownloader::connectRuleFeedSlots()
+{
+    qDebug() << Q_FUNC_INFO << "Connecting rule and feed slots";
+    connect(ui->listRules, SIGNAL(itemSelectionChanged()), this, SLOT(updateRuleDefinitionBox()), Qt::UniqueConnection);
+    connect(ui->listRules, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleRuleCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+    connect(ui->listFeeds, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleFeedCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+}
```

Comment:
The `this` pointer isn't needed here. (And below)
Also reading the description [here](https://doc.qt.io/qt-5/qt.html#ConnectionType-enum) I think it is better to do `Qt::UniqueConnection | Qt::AutoConnection` instead.